runtime/doc/lsp.txt

```diff
@@ -425,6 +425,13 @@ therefore the linked highlights), use the |:highlight| command: >
     highlight LspDiagnosticsDefaultError guifg="BrightRed"
 <
 
+Note: Most custom colorschemes call `highlight clear` by convention on
+initialization. If the colorscheme does not define highlight groups for LSP
+diagnostics, you can set can set the highlights after the colorscheme has
+initialized with an autocommand: >
```

Comment:
I think it would be better to specify that `autocommand` needs to be called before setting `colorshcme` as follows.

If the colorscheme does not define highlight groups for LSP diagnostics, you can set the highlights after the colorscheme has initialized with an autocommand (You will need to call autocommand before setting up colorschme):

Reply:
Better?